tsconfig.json

```diff
@@ -35,7 +35,6 @@
         "src/ipywidgets",
         "src/smoke",
         "src/test/datascience/extensionapi",
-        "src/test/web",
```

Comment:
For better development experience (i.e. during editing we'll get code completions)